Client/Frontend/Home/Wallpapers/v2/NetworkServices/WallpaperImageLoader.swift

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/*
+ TODO: https://mozilla-hub.atlassian.net/browse/FXIOS-4423
+ These are brief sketches for what the image loader might look like as I was
+ experimenting with async/await. The proper implementation will be done in the
+ linked ticket.
+*/
+class WallpaperImageLoader {
```

Comment:
Protocol?

Reply:
Like I said in the comment, this isn't the actual implementation for the image loader. I'll clean this up when I get to the images PR. For now, this was just me playing with ideas to get to grips with how to do networking properly with async/await. I don't think it's harmful to leave this in the code because a) I have to get to it anyway, and build on this; and, b) it's not being used anywhere, so there's no danger.

I can remove it, though, if you'd like me to. Lemme know.